src/main/config/services/edu.gemini.aspen.gmp.services.properties.SimplePropertyHolder-default.cfg

```diff
@@ -1,7 +1,8 @@
 #    Contains the description of the properties that can be obtained through the
 #    GIAPI. Each property has a string value associated.
 
-GMP_HOST_NAME = localhost
+#GMP_HOST_NAME = localhost
+GMP_HOST_NAME = 172.18.13.80
```

Comment:
This is the one we use for local development,, do we want it to point to IG2?

Reply:
No, good point. I am going to change it. Thanks!!!